src/main/java/de/thomas_oster/liblasercut/drivers/ThunderLaserProperty.java

```diff
@@ -0,0 +1,189 @@
+/**
+ * This file is part of LibLaserCut.
+ * Copyright (C) 2011 - 2014 Thomas Oster <mail@thomas-oster.de>
+ * Copyright (c) 2018 Klaus Kämpf <kkaempf@suse.de>
+ *
+ * LibLaserCut is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LibLaserCut is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with LibLaserCut. If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+package de.thomas_oster.liblasercut.drivers;
+
+import de.thomas_oster.liblasercut.FloatPowerSpeedFocusProperty;
+import java.util.Arrays;
+
+/**
+ *
+ * @author kkaempf
+ *
+ * It's all pretty ugly. This should rather extend PowerSpeedFocusFrequencyProperty.
+ * But this is incompatible with RasterizableJobPart(), which is expecting a FloatPowerSpeedFocusProperty :-/
+ */
+public class ThunderLaserProperty extends FloatPowerSpeedFocusProperty {
+
+  private int min_power = 10;
+
+  public ThunderLaserProperty()
+  {
+  }
+
+  /**
+   * Get the value of min power
+   *
+   * @return int
+   */
+  public int getMinPower()
+  {
+    return min_power;
+  }
+
+  /**
+   * Set the value of min power
+   *
+   * @param new min power
+   */
+  public void setMinPower(int power)
+  {
+    this.min_power = power;
+  }
+
+  private int speed = 100;
```

Comment:
Why do you have an extra speed field if the superclass already has one?
With the current solution, the serialization of the settings looks a bit weird because it containts two entries:
```
    <speed defined-in="com.t_oster.liblasercut.FloatPowerSpeedFocusProperty">100.0</speed>
    <speed>100</speed>
...
```

Reply:
Argh. Removed.